src/assets/server/rest/v1/schemas/site-area/site-area-create.json

```diff
@@ -19,14 +19,19 @@
       "$ref": "site-area#/definitions/voltage"
     },
     "smartCharging": {
-      "$ref": "site-area#/definitions/smartCharging"
+      "$ref": "site-area#/definitions/smartCharging",
+      "default": false
```

Comment:
The reason for the default value is, that the smart charging property always had a default, when we used the security classes. (filterBoolean-Method)
So when the property was not set in the request, smart charging was set to false. So every existing site area has the property set.
If we just leave the property, because not mandatory, this causes errors.

For example: One site area is already saved with the property set to false and a new one without the property. 
In this case the check for the consistency of the tree fails. 
As a result you would not be able to add new site area as parents to existing site areas, which already have the property.